ceph-pr-clang-tidy/build/build

```diff
@@ -0,0 +1 @@
+sudo apt-get install -y clang-tidy
```

Comment:
of course this only works if the build machine is an apt-based system, which it may not be using your current node labels.  It might be more appropriate to install the package inside the pbuilder image in that case, too, although it probably doesn't matter too much; there are ways of adding extra packages to the pbuilder image

Reply:
We still need an answer for this; does clang-tidy exist for RHEL-like systems?  or do we want to restrict this job to only apt systems?

Reply:
It is available but for now lets keep the job to apt systems only

Reply:
How is this accomplished? won't the command be executed (and fail) on all systems?

Reply:
The job specifies the labels for the jenkins builder it wants; we'll need to add something like 'jammy' as a label, that should suffice

Reply:
Where should I add this 'jammy' label?

Reply:
job.node is the list of labels the builder must satisfy.  You probably want jammy && small

Reply:
> job.node is the list of labels the builder must satisfy. You probably want jammy && small



@dmick I have added `node: jammy && small`